config/service_qos.go

```diff
@@ -37,81 +39,104 @@ func init() {
 	for k, v := range shannonQoSTypes {
 		ServiceQoSTypes[k] = v
 	}
+	for k := range shannonEVMChainIDs {
+		ServiceQoSTypes[k] = ServiceIDEVM
+	}
+
 	for k, v := range morseQoSTypes {
 		ServiceQoSTypes[k] = v
 	}
-	for k, v := range shannonTestQoSTypes {
-		ServiceQoSTypes[k] = v
+	for k := range morseEVMChainIDs {
+		ServiceQoSTypes[k] = ServiceIDEVM
 	}
 }
 
+// GetEVMChainID returns the hexadecimal EVM chain ID for a given service ID.
+// If the service ID is not found in the ShannonEVMChainIDs or MorseEVMChainIDs
+// maps, the default EVM chain ID of `0x1` is returned.
+func GetEVMChainID(serviceID protocol.ServiceID) string {
+	if chainID, ok := shannonEVMChainIDs[serviceID]; ok {
+		return chainID
+	}
+	if chainID, ok := morseEVMChainIDs[serviceID]; ok {
+		return chainID
+	}
+	return "0x1" // If not found in either map, return the default EVM chain ID (ETH Mainnet - 1)
```

Comment:
Can you move this into a local constant with a TODO_TECHDEBT to figure out what it should be longer term?

Update the comment too.